### PR TITLE
Added base page and navigation

### PR DESCRIPTION
**Pull request by PeteCoward**

### CODE REVIEW

File: pykafe/pykafe/static/css/pykafe.css [hunk 1]
```diff
@@ -19,6 +19,11 @@ body {
     margin-bottom: 80px;
 }
 
+.block-paragraph {
+    margin-top: 80px;
+    margin-left: 80px;
+    margin-right: 80px;
+
```
Comment (marobo) [inline pykafe/pykafe/static/css/pykafe.css]: please close the bracket

Comment (MarioAP) [reply pykafe/pykafe/static/css/pykafe.css]: done

File: pykafe/pykafe/static/css/pykafe.css [hunk 2]
```diff
@@ -260,4 +266,13 @@ button {
 /* HEADER */
 .header {
     background-color: var(--blue-1);
+	border-bottom: none;
```
Comment (MarioAP) [inline pykafe/pykafe/static/css/pykafe.css]: please correct the indentation

Comment (marobo) [reply pykafe/pykafe/static/css/pykafe.css]: Done!

**Pull request merged**